src/PresentationalComponents/PathwaysTable/PathwaysTable.cy.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { IntlProvider } from '@redhat-cloud-services/frontend-components-translations/';
+import { getStore } from '../../Store';
+import PathwaysTable from './PathwaysTable';
+import fixtures from '../../../cypress/fixtures/pathways.json';
+import { pathwaysTableColumns } from '../../../cypress/support/globals';
+import _ from 'lodash';
+
+// eslint-disable-next-line rulesdir/disallow-fec-relative-imports
+import {
+  TOOLBAR,
+  checkTableHeaders,
+  tableIsSortedBy,
+  checkPaginationTotal,
+  checkPaginationValues,
+  changePagination,
+  PAGINATION_VALUES,
+  SORTING_ORDERS,
+} from '@redhat-cloud-services/frontend-components-utilities';
+
+const ROOT = 'table[aria-label="pathways-table"]';
+const TABLE_HEADERS = _.map(pathwaysTableColumns, (it) => it.title);
+const DEFAULT_ROW_COUNT = 4;
+
+const mountComponent = () => {
+  let activeTab = 1;
+  const store = getStore();
+  cy.mount(
+    <MemoryRouter>
+      <IntlProvider locale={navigator.language.slice(0, 2)}>
+        <Provider store={store}>
+          <PathwaysTable isTabActive={activeTab} />
+        </Provider>
+      </IntlProvider>
+    </MemoryRouter>
+  );
+};
+describe('defaults', () => {
+  beforeEach(() => {
+    cy.intercept('*', {
+      statusCode: 201,
+      body: {
+        ...fixtures,
+      },
+    }).as('call');
+    mountComponent();
+  });
+  it('The Rules table renders', () => {
+    cy.get(ROOT).should('have.length', 1);
+  });
+  it('renders toolbar', () => {
+    cy.get(TOOLBAR).should('have.length', 1);
+  });
+  it('renders table header', () => {
+    checkTableHeaders(TABLE_HEADERS);
+  });
+});
+
+describe('defaults', () => {
+  beforeEach(() => {
+    cy.intercept('*', {
+      statusCode: 201,
+      body: {
+        ...fixtures,
+      },
+    }).as('call');
+    mountComponent();
+  });
+  it(`pagination is set to ${DEFAULT_ROW_COUNT}`, () => {
+    cy.get('.pf-c-options-menu__toggle-text')
+      .find('b')
+      .eq(0)
+      .should('have.text', `1 - ${DEFAULT_ROW_COUNT}`);
```

Comment:
This is not really correct from my point of view. The field `1 - ...` just shows which range of rows is shown currently in the table. So given you have only 4 items in your mock data, this will result in `1 - 4`, however the default pagination is set to 20. Soo you either add more items to your mock data or rewrite the test.

Reply:
I think the naming was a bit confusing and incorrect. I renamed it to illustrate the test better
I added 12 more objects to the fixtures
DEFAULT_ROW_COUNT is ROWS_SHOWN now and it equals the "fixtures.data.length"